stdlib/public/Python/Python.swift

```diff
@@ -1253,6 +1253,24 @@ extension PythonObject : MutableCollection {
   }
 }
 
+extension PythonObject : Sequence {
+  public struct Iterator : IteratorProtocol {
+    let pyObjIterator: PythonObject
```

Comment:
Please rename this to something with less abbreviation. How about `pythonIterator`?
```suggestion
    let pythonIterator: PythonObject
```